code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1521,7 +1521,7 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 		src << "<font color='red'><b>"+pick("The pain is excrutiating!", "Please, just end the pain!", "Your whole body is going numb!")
 
 	if(shock_stage >=60)
-		if(shock_stage == 60) emote("me",1,"'s body becomes limp.")
+		if(shock_stage == 60) custom_emote(1,"'s body becomes limp.")

```

Comment:
Alright, this'll be a problem; Because it uses `'s`, it definately doesn't want a space between their name and `'s`. However, `custom_emote` automatically shoves a space in, [here](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/emote.dm#L32).
I would recommend moving the space adding up below `if(!message)`

Something like:
```
	if(!message)
		input = sanitize(copytext(input(src,"Choose an emote to display.") as text|null,1,MAX_MESSAGE_LEN))
	    input = " [input]"
    else
		input = message
	if(input)
		message = "<B>[src]</B>[input]"
	else
		return
```